### PR TITLE
checkpoint: add dashboard

### LINKED ISSUE #5111: [Check Point] Add Dashboard
Our Check Point integration only includes an ingest pipeline with no additional content. Beyond parsing and map, users expect the ability to visualise the data they're ingested to quickly gain insights. Inline with our other firewall integrations, we need to include a dashboard with the integration. 

We can take inspiration from some of our existing dashboards, but some possible resources to include on the dashboard (depending on Check Point event content):

- Event Volume
- Top Destinations 
- Top Sources
- Protocol Types
- Application Categories
- Events by Type
- Rule Actions
- Top Threats by Name

### PR DESCRIPTION
**Pull request by efd6**
## What does this PR do?

Adds a set of four dashboards:
- Overview
- IP Addresses and Ports
- Time and Traffic
- Time and Place

(The last two are time progressive views on traffic characteristics and geo/org endpoints).

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [x] To be merged after #5440 to allow users to gain R81 support without requiring an upgrade to 8.6 that this change requires.

## Related issues

- Closes #5111

### Overview
overview

### IP Addresses and Ports
addresses_and_ports

### Time and Traffic
time_and_traffic

### Time and Place
time_and_place

### CODE REVIEW

**Comment by efd6** [comment]
I am thinking that in the IP Addressess and Ports dash, "other" should be not shown since in real-world cases the vast majority of IPs and ports will not be in the top 10. The trade-off is that the drill-down loses visibility into that.

File: packages/checkpoint/manifest.yml [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
+version: "1.17.0"
 release: ga
 description: Collect logs from Check Point with Elastic Agent.
 type: integration
 format_version: 1.0.0
 license: basic
 categories: [security]
 conditions:
-  kibana.version: "^7.16.0 || ^8.0.0"
+  kibana.version: "^8.6.0"
```
Comment (kcreddy) [inline packages/checkpoint/manifest.yml:11]: Does this make `7.17` unsupported? Is that okay since it hasn't reached EOL?

**Review by kcreddy — APPROVED** [review]
Minor clarification. LGTM 👍🏼 
Great visualisations 🥇

File: packages/checkpoint/manifest.yml (hunk 1 above)
Comment (efd6) [reply packages/checkpoint/manifest.yml:11]: @jamiehynds What is your view?

Comment (efd6) [reply packages/checkpoint/manifest.yml:11]: Yes, that's correct. Thanks.

**Pull request merged**